<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-collection-of-music-in-a-music-library-application-56.ico</normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-collection-of-music-in-a-music-library-application-56.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 88, 5) ;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pre_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(61, 61, 61);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-fast-forwarding-the-music-on-a-music-application-56 - Copy.png</normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-fast-forwarding-the-music-on-a-music-application-56 - Copy.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(61, 61, 61);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-stop-squared-56.png</normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-stop-squared-56.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="play_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(61, 61, 61);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-android-standalone-google-music-service-for-smartphone-and-other-devices-56.png</normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-android-standalone-google-music-service-for-smartphone-and-other-devices-56.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(61, 61, 61);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-pause-squared-56.png</normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-pause-squared-56.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="next_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(61, 61, 61);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-fast-forwarding-the-music-on-a-music-application-56.png</normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-fast-forwarding-the-music-on-a-music-application-56.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="directory_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(61, 61, 61);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-music-file-stored-on-a-folder-for-playback-56.png</normaloff>E:/Lessons/Python/GUI/Pics/Player/icons8-music-file-stored-on-a-folder-for-playback-56.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="slider_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #fff, stop: 1 black);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 rgb(0, 0, 0), stop: 1 rgb(255, 88, 5));
border: 1px solid black;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: white;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 black, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 black, stop:1 rgb(255, 88, 5));
border: 1px solid #444;
border-radius: 4px;
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Britannic Bold</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid rgb(61, 61, 61);
border-radius: 10px;
color: rgb(61, 61, 61);</string>
    </property>
    <property name="text">
     <string>Song Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Broadway</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid rgb(61, 61, 61);
border-radius: 10px;
color: rgb(61, 61, 61);
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="volume_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Britannic Bold</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(61, 61, 61);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="volume_icon_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(61, 61, 61);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>E:/Lessons/Python/GUI/Pics/Player/icons8-medium-sound-setting-for-.png</pixmap>
    </property>
   </widget>
   <zorder>pre_btn</zorder>
   <zorder>stop_btn</zorder>
   <zorder>play_btn</zorder>
   <zorder>pause_btn</zorder>
   <zorder>next_btn</zorder>
   <zorder>directory_btn</zorder>
   <zorder>name_label</zorder>
   <zorder>time_label</zorder>
   <zorder>volume_label</zorder>
   <zorder>volume_icon_label</zorder>
   <zorder>slider_btn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
